examples_gbs/run_tutorial_similarity.py

```diff
@@ -129,8 +135,8 @@
 
 ##############################################################################
 # Now that we have mastered orbits and events, how can we make a feature vector? It was shown in
-# :cite:`schuld2019quantum` that one way of making a feature vector of a graph is to measure the
-# probabilities of events. Specifically, for a :math:`k` photon event :math:`E_{k, n_{\max}}`
+# :cite:`schuld2019quantum` that one way of making a feature vector of a graph is through the
+# frequencies of events. Specifically, for a :math:`k` photon event :math:`E_{k, n_{\max}}`

```

Comment:
Why this change? The way I see it, the feature vector is defined by the probabilities, which we can estimate through their sample frequency

Reply:
I think Maria was suggesting this because the vector could be of relative frequencies